docker/generate_test_suite.sh

```diff
@@ -16,6 +16,28 @@
 #  See the License for the specific language governing permissions and
 # limitations under the License.
 
+usage="
```

Comment:
Have you tried using this script (generate_test_suite.sh) in EMR?

Reply:
Yes, I tried with generate_test_suite.sh. Currently it's using docker to run the test suites. So only limited functionality is available on cluster from this script.